server/server.js

```diff
@@ -35,6 +35,26 @@ app.get('/api/events', async (req, res) =>{
 
     }
 
+    //post request - adding new events 
+app.post("/api/events", async (req, res) => {
+    try {
+        const newEvent = {
+            title: req.body.title,
+            location: req.body.location,
+            eventtime: req.body.eventtime
+        }
+        const result = await db.query('INSERT INTO events(title, location, eventtime) VALUES ($1, $2, $3) RETURNING *', [newEvent.title, newEvent.location, newEvent.eventtime]);
+        let response = result.rows[0];
+        console.log(response);
```

Comment:
Since we're making changes anyway, (see above comment about React Fragments) let's remove these `console.log` statements from the code we want to merge into `main`.